test/Microsoft.AspNetCore.Mvc.IntegrationTests/BodyValidationIntegrationTests.cs

```diff
@@ -453,7 +453,18 @@ private class Person4
         public async Task FromBodyAndRequiredOnValueTypeProperty_EmptyBody_JsonFormatterAddsModelStateError()
         {
             // Arrange
-            var parameterBinder = ModelBindingTestHelper.GetParameterBinder();
+            var testContext = ModelBindingTestHelper.GetTestContext(
+                request =>
+                {
+                    request.Body = new MemoryStream(Encoding.UTF8.GetBytes(string.Empty));
+                    request.ContentType = "application/json";
+                });
+
+            // Override the AllowInputFormatterExceptionMessages setting ModelBindingTestHelper chooses.
+            var options = testContext.GetService<IOptions<MvcJsonOptions>>().Value;
+            options.AllowInputFormatterExceptionMessages = false;
```

Comment:
This and `FromBodyWithInvalidPropertyData_JsonFormatterAddsModelError()` were explicitly testing `2_0` behaviour. If we don't need that (e.g. we have enough non-integration tests covering these cases), LMK. Can switch these to test `2_1` behaviour.

Reply:
I think it would be good to have to some integration tests that cover the 2.0 behavior. Seems good to leave them if it was an explicit choice.